angio-frontend/src/modules/user/components/ResetPasswordForm.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+    <BaseSingleFormContainer v-bind:title="$t('user.component.resetPassword.title')">
+        <template slot="form">
+            <v-form
+                    v-model="valid"
+                    v-on:submit.prevent="submit"
+                    ref="form"
+                    id="reset-password-form"
+            >
+                <v-card-text class="caption">{{ $t('user.component.resetPassword.content') }}</v-card-text>
+                <v-text-field
+                        v-model="email"
+                        v-bind:label="$t('user.component.resetPassword.email.field')"
+                        v-bind:rules="[v => !!v || $t('user.component.resetPassword.email.validation.NotEmpty')]"
+                        v-bind:disabled="fetching"
+                        type="text"
+                        name="email"
+                        prepend-inner-icon="person"
+                        browser-autocomplete="off"
+                        outline
+                        required
+                ></v-text-field>
+                <BuiltInErrorMessage
+                        v-bind:error-messages="errorMessages.map(message => $t(message))"
```

Comment:
Move to computed: [vue ts computed](https://github.com/vuejs/vue-class-component), [ts class getters](https://www.typescriptlang.org/docs/handbook/classes.html)

Reply:
Done in: LoginForm, ResetPasswordForm, AccountActivationForm